src/ert/services/_storage_main.py

```diff
@@ -25,6 +26,11 @@
 from ert.shared import find_available_socket
 from ert.shared.storage.command import add_parser_options
 from ert.trace import tracer
+from everest.detached.everserver import (
```

Comment:
There is also `ert.shared.get_machine_name`. Are they the same?
(there should be some code for certificates also in ert namespace)

Reply:
The one in `net_utils` appears to have been copied from everest at some point, will consolidate 👍